modules/py_csi.c

```diff
@@ -194,12 +194,13 @@ static mp_obj_t py_omv_csi_height() {
 static MP_DEFINE_CONST_FUN_OBJ_0(py_omv_csi_height_obj, py_omv_csi_height);
 
 static mp_obj_t py_omv_csi_get_fb() {
-    if (framebuffer_get_depth() < 0) {
+    image_t image;
+
+    if (framebuffer_get_depth(csi.fb) < 0) {
```

Comment:
Shouldn't this use `framebuffer_t *fb = framebuffer_get(0);` so you can specify the frame buffer to get later on?